docs/bnb_greenfield/guide/core-concept/cross-chain/contract-list.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Contract Entrypoint
+order: 7
+---
+
+# Contract Entrypoint
```

Comment:
Because it is under `core-concept`, while this is a very detailed network info, can we consider put it some-where else.

Reply:
Can we put it under the `get started`, at the bottom of that directory?

Reply:
Done